tamr_client/categorization/project.py

```diff
@@ -21,6 +25,33 @@ def _from_json(url: URL, data: JsonDict) -> CategorizationProject:
     )
 
 
+def create(
+    session: Session,
+    instance: Instance,
+    name: str,
+    description: Optional[str] = None,
+    external_id: Optional[str] = None,
+    unified_dataset_name: Optional[str] = None,
+) -> Project:
+    """Create a Categorization project in Tamr
+    Args:
```

Comment:
```suggestion
    """Create a Categorization project in Tamr
    
    Args:
```

The empty line is needed here for the resulting docs to be formatted properly.  This needs to be changed in the other project creation function docstrings, and in that of `manual_labels` below.

Reply:
updated

Reply:
Will create a separate issue for the testing portion